docs/misc/third-parties.md

```diff
@@ -0,0 +1,36 @@
+# List of third parties
+
+That need to be notified when a protocol breaking change occurs.
+
+### Using trezorlib:
+_This usually requires some code changes in the affected software._
+
+- Electrum https://github.com/spesmilo/electrum
+- HWI https://github.com/bitcoin-core/HWI
+- Trezor Agent https://github.com/romanz/trezor-agent
+
+### Using no Trezor libraries
+- Monero https://github.com/monero-project/monero
+- Mycelium Android https://github.com/mycelium-com/wallet-android
+- Mycelium iOS https://github.com/mycelium-com/wallet-ios
+- Blockstream Green Android https://github.com/Blockstream/green_android
+- Blockstream Green iOS https://github.com/Blockstream/green_ios
+
+### Using Connect:
+_This usually only requires updating Connect to the latest version._
+
```

Comment:
Let's add a link to https://github.com/trezor/connect/network/dependents

Reply:
Added.